### ISSUE TITLE
Bump rollup peer dependency to support v2

### ISSUE THREAD

**Issue by vladshcherbin** [tivac/modular-css#776]
👋 Hey, I've tried to install current latest version for rollup (25.8.0) and it gave me a warning:

```
warning " > @modular-css/rollup@25.8.0" has incorrect peer dependency "rollup@^1.1.2".
```

I found recent https://github.com/tivac/modular-css/pull/733 about rollup 2. Would be awesome to see this warning go away 🙌 

## Expected Behavior

No warnings on installation.

## Current Behavior

Warning during installation.

## Possible Solution

Bump rollup peer dependency.

## Steps to Reproduce (for bugs)

1. Create new project
2. Add rollup >= 2
3. Install latest plugin version

## Context

Tried to switch from css modules and saw a welcome warning 😢 

## Your Environment

| Executable | Version |
| ---: | :--- |
| `modular-css`  | 25.8.0 |
| `yarn --version`  | 1.22.4 |
| `node --version` | 14.0.0 |

| OS | Version |
| --- | --- |
| macOS Mojave | 10.14.6 |

**Comment by tivac** [comment]
Whoops 🤦🏻‍♂️. Thanks for the report! A PR would be great, if not I'll try to get to this soon.

**Comment by tivac** [comment]
Fixed in `@modular-css/rollup@25.8.1` (also fixed/published `@modular-css/rollup-rewriter@25.8.1` while I was at it)

**Comment by vladshcherbin** [comment]
@tivac thanks 🎉
